lib/geordi/cucumber.rb

```diff
@@ -8,9 +8,17 @@
 
 module Geordi
   class Cucumber
+    def run(cucumber_options, options = {})
+      cucumber_options.map! do |option|
+        if option.match?(/\S\s\S/)
+          # put options with a whitespace in quotes. Relevant for negated tag options ('not @tag')
+          "'#{option}'"
+        else
+          option
+        end
+      end
+      self.argv = cucumber_options
```

Comment:
This could be simplified to 
```suggestion
      self.argv = arguments.each do |arg|
        # Ensure arguments containing white space are kept together
        arg.replace %('#{arg}') if arg.match?(/\S\s\S/)
      end
```

Reply:
Unfortunately this does not work. `#each` simply returns the original array . But I can shorten the code to  
```suggestion
      self.argv = arguments.map do |arg|
        # Ensure arguments containing white space are kept together
        arg.match?(/\S\s\S/) ? %('#{arg}') : arg
      end
```